lib/smart_proxy_ansible/plugin.rb

```diff
@@ -3,22 +3,20 @@ module Ansible
     # Calls for the smart-proxy API to register the plugin
     class Plugin < Proxy::Plugin
       http_rackup_path File.expand_path('http_config.ru',
-                                        File.expand_path('../', __FILE__))
+        File.expand_path(__dir__))
       https_rackup_path File.expand_path('http_config.ru',
-                                         File.expand_path('../', __FILE__))
+        File.expand_path(__dir__))
```

Comment:
I also think the double call to `File.expand_path` is redundant.

If you rely on Smart Proxy 2.3 or newer, you can use `rackup_path File.expand_path('http_config.ru', __dir__)` instead of the two separate calls.

Reply:
I've got no idea if it's safe to assume that and the only dependency is development dependency :thinking:
Although it would be nice to fix it, I don't think it should be part of this :thinking:

Reply:
Agreed, but just wanted you to know. You can at least simplify it further with `https_rackup_path File.expand_path('http_config.ru', __dir__)`. The extra `File.expand_path` doesn't really add anything.

Reply:
I've put it on one line, and I'd like to postpone it after the merge of this.

Reply:
But lets have a PR for it not to forget: #25 :)